src/main/java/frc/robot/commands/AlignCommand.java

```diff
@@ -11,9 +11,9 @@
 
     private static final double Ki = 0.00200;
     private static final double Kd = 0.000004;
-    private static double Kp = 0.017000;
+    private static double Kp = 0.01700;
 
-    protected final PIDController pid = new PIDController(Kp, Ki, Kd);
+    protected PIDController pid = new PIDController(Kp, Ki, Kd);
```

Comment:
why not keep this as final and just call `setP`, etc

Reply:
Good point. Wait one second, I'm going to try to push a few edits for this branch after rebasing it with the main one.